hamill/src/main/java/com/codesquad/datastructurestudy/queue/ArrayQueue.java

```diff
@@ -0,0 +1,71 @@
+package com.codesquad.datastructurestudy.queue;
+
+class Queue {
+    private int maxSize;
+    private int[] queueArray;
+    private int front;
+    private int rear;
+    private int currentSize;
+
+    public Queue(int size) {
+        this.maxSize = size;
+        this.queueArray = new int[size];
+        front = 0;
+        rear = -1;
+        currentSize = 0;
+    }
+
+    public void insert(int item) {
+        // check if queue is already full
+        if (isQueueFull()) {
+            System.out.println("Queue is full!!");
+            return;
+        }
+        // for wrapping the queue in case
+        // max size has reached
+        if (rear == maxSize - 1) {
+            rear = -1;
+        }
+        // increment rear then insert item
+        queueArray[++rear] = item;
+        currentSize++;
+        System.out.println("Added to queue " + item);
+    }
+
+    public int remove() {
+        // check if queue is empty
+        if (isQueueEmpty()) {
+            throw new RuntimeException("Queue is empty");
+        }
+        // System.out.println("front= " + front + " maxSize= " + maxSize);
+        // retrieve item then increment
```

Comment:
불필요한 소스 주석은 제거하는게 좋을 것 같아요!
```suggestion
        // retrieve item then increment
```